docs/site/todo-list-tutorial-sqldb.md

```diff
@@ -0,0 +1,134 @@
+---
+lang: en
+title: 'Running on relational database'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-sqldb.html
+summary:
+  LoopBack 4 TodoList Application Tutorial - Running on Relational Database
+---
+
+If you are running this example using a relational database, there are extra
+steps in order to set up constraints in the database. We're using PostgreSQL for
+illustration, it would work for other relational databases in a similar way.
+
+### Create a new datasource
+
+We're going to reuse the DataSource `db` to connect to the PostgreSQL database
+instead of the in-memory database.
+
+First, remove the existing `db`, run:
+
+```sh
+rm src/datasources/db.datasource.*
+```
+
+Then, create the datasource with the same name with the `lb4 datasource` command
+and select `PostgreSQL` as the connector.
+
+```sh
+$ lb4 datasource
+? Datasource name: db
+? Select the connector for db:
+  Redis key-value connector (supported by StrongLoop)
+  MongoDB (supported by StrongLoop)
+  MySQL (supported by StrongLoop)
+❯ PostgreSQL (supported by StrongLoop)
+  Oracle (supported by StrongLoop)
+  Microsoft SQL (supported by StrongLoop)
+  REST services (supported by StrongLoop)
+  ...
+```
+
+### Specify the foreign key constraint in the models
+
+Based on the model relations we've specified in the 3 models, the expected
+database tables look like below:
+![database tables](todo-list-tutorial-dbtables.png)
+
+{% include note.html content="
+There is an ongoing work on supporting strong relation with referential integrity.  For details, please see [epic #2231](https://github.com/strongloop/loopback-next/issues/2331).
```

Comment:
Nitpick: `There is an ongoing work` -> `There is ongoing work`, `relation` -> `relations`